src/ontoweaver/__init__.py

```diff
@@ -136,3 +138,39 @@ def reconciliate_write(nodes: list[Tuple], edges: list[Tuple], biocypher_config_
     import_file = bc.write_import_call()
 
     return import_file
+
+def validate_only(data_mappings: dict):
```

Comment:
I would go for a more explicit name:
```suggestion
def validate_input_data(data_mappings: dict):
```

Reply:
I also think we should follow the same approach than the one we discussed in PR #82 : have a function that takes filenames, handle the loading, and then call another function that takes loaded objects and do the job.